ui/src/app/components/elements/list.component.ts

```diff
@@ -518,4 +518,23 @@ export class ElementsListComponent extends BaseComponent implements OnInit {
   humanizedDate(date) {
     return moment.duration(moment().diff(date)).humanize() + ' ago';
   }
+
+  search(term?: string) {
+    if (term) {
+      if (!this.query?.includes('workspaceVersionId')) {
```

Comment:
plz move the default query strings on initialization part, since search handler execution frequency is high